### PR TITLE
New tribe members to website

### PR DESCRIPTION
**Pull request by memsharded**
@datalogics-robb
@boussaffawalid 
@sanblch 
@raplonu
@roalter 
@sturmf

If you could please do a quick check that your information is correct. Thanks again!

### CODE REVIEW

File: src/js/3_tribe-members.js [hunk 1]
```diff
@@ -383,4 +383,45 @@ var tribeMembers = [
...
+  {
+    name: 'Luis Roalter',
+    image: 'luis_roalter.jpg',
+    description: "I'm a C++ and Python developer since 2014. Main topics are continuous integration and deployment and tool development. Trying to solved everything with Python and Git.",
+    company: 'Rohde & Schwarz',
+    twitter: 'https://twitter.com/roalter'
+  },
+  {
+    name: 'Fabian Sturm',
+    image: 'luis_roalter.jpg',
```
Comment (sturmf) [inline src/js/3_tribe-members.js:421]:
```suggestion
    image: 'fabian_sturm.jpg',
```

File: src/js/3_tribe-members.js [hunk 2]
```diff
@@ -383,4 +383,45 @@ var tribeMembers = [
...
+    description: "I am a long time software developer and project lead at Rohde & Schwarz, with experience in C++ and Python. My professional interests are new software stacks and programming languages and in my private time I participate in sailing races.",
+    company: 'Rohde & Schwarz',
+    twitter: 'https://twitter.com/roalter'
```
Comment (sturmf) [inline src/js/3_tribe-members.js:424]:
```suggestion
```

**Review by sturmf — CHANGES_REQUESTED** [review]
It seems Louis data got copied into mine, I am unfortunately also not active on twitter.

**Comment by sturmf** [comment]
Thanks, lgtm now.

**Review by datalogics-robb — APPROVED** [review]

**Pull request merged**